compiler/rustc_query_system/src/ich/impls_syntax.rs

```diff
@@ -115,7 +115,7 @@ impl<'tcx> HashStable<StableHashingContext<'tcx>> for rustc_feature::Features {
         self.enabled_lang_features().hash_stable(hcx, hasher);
         self.enabled_lib_features().hash_stable(hcx, hasher);
 
-        self.all_lang_features()[..].hash_stable(hcx, hasher);
+        // FIXME: why do we hash something that is a compile-time constant?
```

Comment:
The comment refers to the loop that follows it.

The change here is to no longer hash `all_lang_features` because that does not exist any more. That used to return an array with all the `bool` fields, but those fields are gone now. So I don't think there is anything I can undo here.

Reply:
👍